fastlane/Fastfile

```diff
@@ -23,48 +23,48 @@ before_all do
     UI.user_error!("~/.simplenoteios-env.default not found: Please copy env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values")
   end
   unless File.file?(PROJECT_ENV_FILE_PATH)
-    UI.user_error!("project.env not found: Make sure your configuration is up to date with `rake dependencies`")
+    UI.user_error!('project.env not found: Make sure your configuration is up to date with `rake dependencies`')
   end
 
   setup_ci
 end
 
-error do |lane, exception, options|
+error do |lane, _exception, _options|
   deoccupy_test_account if lane == :run_ui_tests
 end
 
 platform :ios do
-########################################################################
-# Environment
-########################################################################
-PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
-VERSION_FILE_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
-OUTPUT_DIRECTORY_PATH = File.join(PROJECT_ROOT_FOLDER, 'build', 'results')
-
-Dotenv.load(USER_ENV_FILE_PATH)
-Dotenv.load(PROJECT_ENV_FILE_PATH)
-ENV[GHHELPER_REPO="Automattic/simplenote-ios"]
-ENV["PROJECT_NAME"]="Simplenote"
-ENV["PROJECT_ROOT_FOLDER"]=PROJECT_ROOT_FOLDER
-ENV['APP_STORE_STRINGS_FILE_NAME']='AppStoreStrings.pot'
-ENV["PUBLIC_CONFIG_FILE"]=VERSION_FILE_PATH
-ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
-ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'
-REPOSITORY_NAME="simplenote-ios"
-APP_STORE_BUNDLE_IDENTIFIER = 'com.codality.NotationalFlow'.freeze
-
-# Use this instead of getting values from ENV directly
-# It will throw an error if the requested value is missing
-def get_required_env(key)
-  unless ENV.key?(key)
-    UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?")
+  ########################################################################
+  # Environment
+  ########################################################################
+  PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
+  VERSION_FILE_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
+  OUTPUT_DIRECTORY_PATH = File.join(PROJECT_ROOT_FOLDER, 'build', 'results')
+
+  Dotenv.load(USER_ENV_FILE_PATH)
+  Dotenv.load(PROJECT_ENV_FILE_PATH)
+  ENV.fetch(GHHELPER_REPO = 'Automattic/simplenote-ios', nil)
```

Comment:
This might not be the right auto-fix.

The original line has always been a mistake/typo. It was written like this:
```
ENV[GHHELPER_REPO="Automattic/simplenote-ios"]
```
While what we really wanted to do was just:
```
GHHELPER_REPO="Automattic/simplenote-ios"
```
But that previous line was valid, because what it did was to (1) do that `GHHELPER_REPO="Automattic/simplenote-ios"` from the expression within the `[ ]`, which is an expression that assigns the string to the `GHHELPER_REPO` constant… then return its value (unlike e.g. Swift, in Ruby, the `=` operator also returns the value). And then `ENV[…that expression…]` was using the return value of that expression as the key to index `ENV` (so that was trying to do `ENV['Automattic/simplenote-ios']`, which doesn't even make sense for our case…)… and do nothing with the result (i.e. discarded the returned value if any). Complete nonsense 😓 

Sadly we have this annoying typo in the `Fastfile` of most of our repos (yay copy/pasting…) and never fixed it (because… eh… it doesn't break anything, as in the end `GHHELPER_REPO` does end up being assigned the value we want), but we really should.

```suggestion
  GHHELPER_REPO = 'Automattic/simplenote-ios'
```

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/6bde715607693c183e325d091a005d06043691f6